Prox/Prox.xcodeproj/project.pbxproj

```diff
@@ -353,11 +353,11 @@
 		7B5095631DB531CA007C54F0 /* Models */ = {
 			isa = PBXGroup;
 			children = (
-				E65857DD1DB69F4200CD18F8 /* ReviewProvider.swift */,
-				7B5095641DB531CA007C54F0 /* Place.swift */,
-				7B48EBD91DD1F679005C36CB /* GeofenceRegion.swift */,
-				7BA05BE41DD4CCB1009AC121 /* Event.swift */,
 				2380CA901DE4140A002D73F1 /* AppState.swift */,
+				7BA05BE41DD4CCB1009AC121 /* Event.swift */,
```

Comment:
Yeah, they might -- it happens all the time :) Generally, they're pretty easy to resolve manually.